### DISCUSSION TITLE
Is WalletConnectSwiftV2 available as a Pod?

### DISCUSSION THREAD

**Discussion by johnnynanjiang** [WalletConnect/walletconnect-monorepo#1607] [Developer Support]
Swift Package didn't work in some cases, especially when network has firewall for some devs.

**Comment by flypaper0** [comment]
@johnnynanjiang what kind of errors do you encounter? We planning to add cocoapods support in next release, will this solve your problem?

**Reply by johnnynanjiang** [reply]
Thanks @flypaper0 , that would be great.

Some of devs have difficulty resolving dependencies via Swift Package due to network issues, or someone might prefer CocoaPod dependency management.
